packages/core/test/invoke.test.ts

```diff
@@ -3357,10 +3358,13 @@ describe('invoke', () => {
     });
 
     const actorRef = createActor(machine).start();
+    actorRef.system.inspect((e) => {
+      e;
+    });
     actorRef.send({ type: 'NEXT' });
-    await sleep(3);
-    expect(actorRef.getSnapshot().status).toBe('done');
-  });
+
+    await waitFor(actorRef, (s) => s.status === 'done');
+  }, 100000);
```

Comment:
I'm pretty sure you have some debugging leftovers here, changes to this file should be reverted